nrf_security/src/drivers/nrf_oberon/CMakeLists.txt

```diff
@@ -18,38 +18,38 @@ if(CONFIG_MBEDTLS_PSA_CRYPTO_C)
       ${NRFXLIB_ROOT}/crypto/nrf_oberon/include/mbedtls
       ${OBERON_PSA_CRYPTO_PATH}/oberon/drivers
   )
-
 endif()
 
-if (COMPILE_PSA_APIS)
-  append_with_prefix(src_crypto_oberon ${OBERON_PSA_CRYPTO_PATH}/oberon/drivers/
-    oberon_aead.c
-    oberon_cipher.c
-    oberon_ecdh.c
-    oberon_ecdsa.c
-    oberon_hash.c
-    oberon_helpers.c
-    oberon_kdf.c
-    oberon_key_pair.c
-    oberon_mac.c
-    oberon_rsa.c
-  )
-
-  append_with_prefix_ifdef(CONFIG_PSA_CRYPTO_DRIVER_ALG_CTR_DRBG_OBERON
-    src_crypto_oberon ${OBERON_PSA_CRYPTO_PATH}/oberon/drivers/
-      oberon_hmac_drbg.c
-  )
+macro(add_oberon_source kconfig filename)
```

Comment:
Use the `_ifdef` convention for the macro name?

Reply:
The macro doesn't follow the _ifdef convention.

It constructs a variable from the kconfig and then ifdef's based on that.

In the _ifdef convention you take the variable as-is and ifdef on it.

Reply:
But I could maybe rename it to something more appropriate?

Like

maybe_add_oberon_source
conditionally_add_oberon_source

?

Reply:
One can argue that the macro is narrow enough in scope that the small breakage on convention isn't as problematic. _ifdef looks very nice as a suffix